tools/print_dependencies.py

```diff
@@ -33,4 +36,33 @@
     })
 
   deps = sorted(deps, key=lambda k: k['identifier'])
-  print json.dumps(deps, sort_keys=True, indent=2)
+
+  # Print all dependencies if a target is unspecified
+  if len(sys.argv) == 1:
+    print(json.dumps(deps, sort_keys=True, indent=2))
+    exit(0)
+
+  # Bazel target to print
+  target = sys.argv[1]
+  output = subprocess.check_output(['bazel', 'query', 'deps(%s)' % target])
+
+  repos = set()
+
+  # Gather the explicit list of repositories
+  repo_regex = re.compile('^@(.*)\/\/')
+  for line in output.split('\n'):
+    match = repo_regex.match(line)
+    if match: repos.add(match.group(1))
```

Comment:
Nit: prefer split lines:
```
if match:
  foo
```
but this is more a personal preference; I haven't seen much Python code like this..